meta/autoid/constid.go

```diff
@@ -0,0 +1,128 @@
+package autoid
+
+import "github.com/pingcap/errors"
+
+// GetSystemSchemaID gets the system schema ID.
+func GetSystemSchemaID(dbName string) (int64, error) {
+	id, ok := SystemSchemaIDMap[dbName]
+	if !ok {
+		return 0, errors.Errorf("get system schema id failed, unknown system schema `%v`", dbName)
+	}
+	return SystemIDFlag | id, nil
+}
+
+// GetInformationSchemaTableID gets the information_schema table ID.
+func GetInformationSchemaTableID(tableName string) (int64, error) {
+	id, ok := InformationSchemaTableIDMap[tableName]
+	if !ok {
+		return 0, errors.Errorf("get system table id failed, unknown system table `%v`", tableName)
+	}
+	return SystemIDFlag | id, nil
+}
+
+// GetPerformanceSchemaTableID gets the performance_schema table ID.
+func GetPerformanceSchemaTableID(tableName string) (int64, error) {
+	id, ok := PerformanceSchemaTableIDMap[tableName]
+	if !ok {
+		return 0, errors.Errorf("get system table id failed, unknown system table `%v`", tableName)
+	}
+	return SystemIDFlag | id, nil
+}
+
+// SystemIDFlag exports for test.
+const SystemIDFlag = 1 << 62
```

Comment:
Why `1 << 62`?

Reply:
We want to use the  highest bit position as a system schema ID flag.